etcdctl/ctlv3/command/version_command.go

```diff
@@ -31,5 +31,6 @@ func NewVersionCommand() *cobra.Command {
 }
 
 func versionCommandFunc(cmd *cobra.Command, args []string) {
-	fmt.Println(version.Version)
+	fmt.Println("etcd version:", version.Version)
+	fmt.Println("API version: 3")
```

Comment:
probably just use version.Version here instead of 3 (or even better - keep major.minor only)